README.rst

```diff
@@ -578,7 +580,8 @@ candidate list further down.
   - Examples::
 
         maven:org.apache.xmlgraphics/batik-anim@1.9.1
-        maven:org.apache.xmlgraphics/batik-anim@1.9.1?packaging=sources
+        maven:org.apache.xmlgraphics/batik-anim@1.9.1?classifier=sources
+        maven:org.apache.xmlgraphics/batik-anim@1.9.1?package=zip&classifier=dist
```

Comment:
Thanks!
Is this really `package` and not `packaging` there? I used that based on this: https://maven.apache.org/pom.html but you should know better than me!

Also do you mind to add a [DCO](https://developercertificate.org/) signed-off-by in the PR text body at least and in the future in commit messages?

Reply:
Let me fix that. Technically I think it should be extension here since packaging is more a term used in the pom which brings in a lifecycle and plugins. The resulting files may not map directly to a package. Example: package maven-plugin still generates a jar. If you're trying to programmatically fetch from the repo and you only have the package, you end up having to embed knowledge of various mappings. Here we are referring to the resulting coordinates to construct a file, and extension is much more obvious.